### PR TITLE
Exam project

### PR DESCRIPTION
**Pull request by oladelmar**

### CODE REVIEW

File: src/components/layouts/Side.vue [hunk 1]
```diff
@@ -0,0 +1,34 @@
+<template>
+  <v-navigation-drawer>
+    <v-list dense class="pt-0">
+      <v-list-tile @click="">
```
Comment (rapi) [inline src/components/layouts/Side.vue]: Dont leave empty @click

File: src/components/layouts/Side.vue [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        <v-list-tile-content>
+          <v-list-tile-title>
+            <router-link to="/" active-class="active">Home</router-link>
```
Comment (rapi) [inline src/components/layouts/Side.vue]: dont leave :to string

File: src/components/layouts/Header.vue [hunk 3]
```diff
@@ -0,0 +1,31 @@
+<template>
+  <div>
+    <v-toolbar>
+      <v-toolbar-title>
+        <router-link to="/">Github</router-link>
+      </v-toolbar-title>
+      <v-toolbar-items class="hidden-sm-and-down">
+        <v-btn flat>
+          <router-link to="/issues">Issues</router-link>
```
Comment (rapi) [inline src/components/layouts/Header.vue]: Dont leave to String

**Review by rapi — COMMENTED** [review]
Good, some small things but in general very well

File: src/components/layouts/Side.vue (hunk 1 above)
Comment (oladelmar) [reply src/components/layouts/Side.vue]: fixed

File: src/components/layouts/Side.vue (hunk 2 above)
Comment (oladelmar) [reply src/components/layouts/Side.vue]: fixed

File: src/components/layouts/Header.vue (hunk 3 above)
Comment (oladelmar) [reply src/components/layouts/Header.vue]: fixed
Thanks)

**Pull request merged**